how-not-to-be-a-dinosaur.md

```diff
@@ -0,0 +1,117 @@
+  
+
+It was 2017 already. I did not know where to start. I didn't dare to ask for help, because of my misplaced pride. I was writing code before some of my new colleagues were even born… and now I should go to these guys and admit I had no clue what I was doing?
+
+But I tried… I tried to keep up with it for a few months and then I was like: "I don't get it! fuck this shit!" 
+
+I have been doing design ánd development 50/50 for all of my carreer, so now it's time to say goodbye to one of these and focus on the other. And since I got só scared of The New Frontend World, I choose design. This was in early 2018.
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_drawing.png)
+
+_Nope. Not me either._
+
+And after a year of doing (UX) design, I found out this was the wrong choice. Years of intensive therapy and running into a black hole that was another potential burnout, I came to the conclusion that design was not for me. I can design, but I can't handle the stress that goes with it. 
+
+And then it was late 2018. I decided to get back to frontend development.
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_dino.png)
+
+_Yes this is me._
+
+## How to get up to speed? 
+
+Now I faced a big challenge. How to get up to speed after being paused for a few years in a world that changed (and still is changing) so fast? First thing to overcome was my aversion for The New Frontend World. I tried to look at React and all new tools with a fresh and unbiased view.
+
+While I was away in design world a lot has changed within our codebase, but also in The Frontend World. The tools we used improved or were replaced with better alternatives. The tools and codebase became more solid and mature.
+
+Then there was news. React Hooks was about to be released. Which made code só much more readable that I started thinking "Hmm, maybe I can do this".  
+
+After this "challenge accepted!" point the hardest obstacle had to be overwon: how to handle the "I have to start all over again" feeling. I felt like I knew nothing. I was Jon Snow.
+
+## Dem feelings...
+
+Who had I been kidding all these years calling myself a senior developer? How emberrassing was it that I had to ask my 12 years younger colleague for help over and over again...  
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_knowsnothing.png)
+
+Some people will think "what's the big deal?", and I am happy for you if you do, but you have to understand that asking help was a big issue for me. And a major cause for my burn out and personality disorder that came with it.
+
+So I did two things: I decided to start learning all these new tools and code, but more importantly I started to ask for help. And the best thing I did: I told my colleagues how I was feeling and how insecure I was about everything. They responded very well to this and this gave me the confidence to go for it.
```

Comment:
```suggestion
So I did two things: I decided to start learning all these new tools and frameworks and more importantly I started to ask for help. But the most important thing I did: I told my colleagues how I was feeling and how insecure I was about everything. They were very supportive and this gave me the confidence to go for it.
```

"They responded very well" is a bit vague, it's nicer if it's clear in what way they (those mysterious colleague guys) specifically responded.

Reply:
👍